pallets/pallet-bonded-coins/README.md

```diff
@@ -1,4 +1,137 @@
+- `Fungibles`: Implementation of creating and managing new fungibles.
+- `CollateralCurrencies`: Implementation to withdraw and deposit collateral currencies.
+
+### Constants
+
+- `MaxDenomination`: The maximum denomination allowed.
+- `DepositPerCurrency`: The deposit required per currency.
+- `BaseDeposit`: The base deposit amount.
+- `MaxStringLength`: The maximum length of strings.
+- `MaxCurrencies`: The maximum number of currencies allowed.
+
+### Origins 
+
+- `DefaultOrigin`: The default origin for operations.
+- `PoolCreateOrigin`: The origin required to create a pool.
+- `ForceOrigin`: The origin with force privileges.
+
+### Feature guided 
+
+- `BenchmarkHelper`: Helper type for benchmarking to calculate asset ids for collateral and bonded currencies
+
+## Life Cycle of a Pool
+
+1. **Creation**:
+   - A pool is created using the `create_pool` function.
+   - The manager specifies the bonding curve, collateral type, currencies, and whether the bonded coins are transferable.
+   - A deposit is taken from the caller, which can be reclaimed once the pool is destroyed.
+
+2. **Refund Process**:
+   - The refund process can be started by the manager using the `start_refund` function.
+   - The refund process can be forced to start using the `force_start_refund` function, which requires force privileges.
+   - Collateral can be refunded to a specific account using the `refund_account` function, based on the owned bonded currency.
+
+3. **Destruction**:
+   - The destruction process can be started by the manager using the `start_destroy` function. This operation will fail if accounts with bonded currencies still exist.
+   - The destruction process can be forced to start using the `force_start_destroy` function, which requires force privileges. This operation will fail if there are still accounts with bonded currencies.
+   - The destruction process is completed using the `finish_destroy` function, which refunds any taken deposits.
+
+## Functions
+
+### Public Functions
+
+- `create_pool`: Creates a new pool with the specified bonding curve, collateral type, and currencies. During the pool creation step, the manager must specify whether the bonded coins are transferable or not. This flag can not be changed later on. The selected denomination is used not only as metadata but also to scale down the existing supply or the amount specified in a mint or burn operation. An deposit is taken from the caller, which can be reclaimed back, once the pool is getting destroyed. 
+- `reset_team`: Resets the managing team of a pool. Only the admin and the freezer can be updated in this operation. 
+- `reset_manager`: Resets the manager of a pool. 
+- `set_lock`: Sets a lock on a pool. Locks specify who is able to mint and burn a bonded currency. After applying the lock, the pool becomes permissioned, and only the manager is able to mint or burn bonded currencies.
+- `unlock`: Unlocks the pool. 
```

Comment:
This can also be done only by the manager, correct?

Reply:
Yes right: [ea7228e](https://github.com/KILTprotocol/kilt-node/pull/807/commits/ea7228e002fa608ba2742d4b7442c92f73994433)